wodan.opam

```diff
@@ -28,8 +28,8 @@ depends: [
   "lwt" {>= "3.1.0"}
   "lwt_ppx"
   "mirage-logs"
-  "mirage-types"
-  "mirage-types-lwt" {>= "3.0.5"}
+  "mirage-types" {<= "3.6.0"}
+  "mirage-types-lwt" {>= "3.0.5" & <= "3.6.0"}
```

Comment:
Since MirageOS 3.7.0 is released, you should, at least, keep in your mind to update `wodan` to new interface - eg. specialization of `type 'a io` to `'a Lwt.t` and specialization of `type buffer` with `Cstruct.t` :+1:.

Reply:
Or do by yourself abstraction and specialization to MirageOS interfaces (which is what we want to do now).

Reply:
Ideed, that's my next commit ;)